src/main/resources/org/jenkinsci/plugins/pipeline/modeldefinition/ModelInterpreter.groovy

```diff
@@ -106,9 +107,10 @@ public class ModelInterpreter implements Serializable {
                                                 firstError = e
                                             }
                                         }
-                                    }
+                                    }.call()
```

Comment:
I have been wondering about all these "return closure and call it directly" approach that is scattered throughout. Why not just run the closure directly inside then method `nodeOrDockerOrNone`? Is it to clean up the stacktraces or what?